frontend/app/js/app.coffee

```diff
@@ -1,6 +1,9 @@
 angular.module('exercism', ['ui.bootstrap'])
 
 $ ->
+  if $('h1, h2, h3').length && document.title == 'exercism.io'
+    document.title = "#{$($('h1, h2, h3').get(0)).text()} - exercism.io"
```

Comment:
This will change the title of homepage (http://exercism.io/) to `Level up your programming skills - exercism.io`. I think it should be remain `exercism.io`. Maybe we can just use `h1` instead of `h1, h2 and h3`.